[glass] std::snprintf() truncation warning

I got a new warning from a GCC 11.1 release build.
```
> Task :glass:compileGlassAppLinuxx86-64ReleaseExecutableGlassAppCpp FAILED
/home/tav/frc/wpilib/allwpilib/glass/src/app/native/cpp/main.cpp: In static member function 'static _Res std::_Function_handler<_Res(_ArgTypes ...), _Functor>::_M_invoke(const std::_Any_data&, _ArgTypes&& ...) [with _Res = void; _Functor = main(int, char**)::<lambda()>; _ArgTypes = {}]':
/home/tav/frc/wpilib/allwpilib/glass/src/app/native/cpp/main.cpp:274:54: error: '###edit' directive output may be truncated writing 7 bytes into a region of size between 1 and 32 [-Werror=format-truncation=]
  274 |       std::snprintf(editLabel, sizeof(editLabel), "%s###edit",
      |                                                      ^~~~~~~
/home/tav/frc/wpilib/allwpilib/glass/src/app/native/cpp/main.cpp:274:20: note: 'snprintf' output between 8 and 39 bytes into a destination of size 32
  274 |       std::snprintf(editLabel, sizeof(editLabel), "%s###edit",
      |       ~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  275 |                     gKeyEdit ? "(press key)" : name);
      |                     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

--- a/glass/src/app/native/cpp/main.cpp
+++ b/glass/src/app/native/cpp/main.cpp
@@ -264,7 +264,7 @@ int main(int argc, char** argv) {
 
       ImGui::Text("Key:");
       ImGui::SameLine();
-      char editLabel[32];
+      char editLabel[40];
       char nameBuf[32];
       const char* name = glfwGetKeyName(enterKey, 0);
       if (!name) {

